test/faraday_rate_limiter_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+require 'airrecord/faraday_rate_limiter'
+
+class FaradayRateLimiterTest < Minitest::Test
+  def setup
+    @stubs = Faraday::Adapter::Test::Stubs.new
+    @rps = 5
+    @sleeps = []
+    @connection = Faraday.new { |builder|
+      builder.request :airrecord_rate_limiter,
+        requests_per_second: @rps,
+        sleeper: ->(s) { @sleeps << s }
```

Comment:
Nice. I wouldn't have thought to make the `sleeper` lambda replaceable via dependency injection. I'll try to remember this next time I'm building something hard to test.

Reply:
If I had `Mocha`, I wouldn't have bothered 🤷‍♂️ But I didn't want to pull in the dependency.